tests/test_forward_one_to_one_relation_crudl_operations_are_supported.py

```
@@ -0,0 +1,164 @@
+    amz_author_profile.refresh_from_db()
+    assert amz_author_profile.author_id == None
+
+
+@pytest.mark.django_db
+def test_getting_relation_with_get_many_should_work(client: Client) -> None:
+    """Test listing relations with GET many (list) request."""
+    author = models.Author.objects.create(
+        name="Some author",
+        birth_date="1990-01-01",
+    )
+    amz_author_profile = models.AmazonAuthorProfile.objects.create(
```

Ruff (0.8.2)

Local variable `amz_author_profile` is assigned to but never used

Remove assignment to unused variable `amz_author_profile`

(F841)